client/src/modules/employees/registry/search.modal.html

```diff
@@ -11,6 +17,9 @@
     </ol>
   </div>
 
+  <uib-tabset>
+      <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+
   <div class="modal-body" style="max-height : 400px; overflow: auto;">
```

Comment:
This indenting is off - can you make go ahead and make sure all this code is properly indented?  It will make it easier to read and modify later.